packages/api-sdk/src/resources/base.ts

```diff
@@ -1,6 +1,7 @@
 import Fetcher, { FetcherOptions } from './fetcher';
 
 export type Fields = string[] | ReadonlyArray<string>;
+export type IDs = string[] | ReadonlyArray<string>;
```

Comment:
```suggestion
export type Fields = readonly string[];
export type IDs = readonly string[];
```
`string[]` extends `readonly string[]`